### ISSUE TITLE
Seperate package for client?

### ISSUE THREAD

**Issue by powellnorma** [collabora/WhisperLive#218]
I think most of the dependencies that get installed with `pip install whisper-live` are only needed for the server, not the client. How can I use the client without installing all the server's packages (torch, triton, ..)?

**Comment by makaveli10** [comment]
@powellnorma you are right, the dependencies installed with pip are combined for both server and client. And currently, there is no way of installing whisper-live from pip only for the client.
